runtests.py

```diff
@@ -327,6 +327,9 @@ def update_cpp17_extension(ext):
     clang_version = get_clang_version(ext.language)
     if clang_version:
         ext.extra_compile_args.append("-std=c++17")
+        if sys.version_info[0] < 3:
+            ext.extra_compile_args.append('-Wno-register')  # the Python 2.7 headers contain register
+                # which clang warns about with C++17 on
```

Comment:
It helps readability if special words stick out of the normal reading flow.
```suggestion
            # The Python 2.7 headers contain the 'register' modifier
            # which clang warns about in C++17 mode.
            ext.extra_compile_args.append('-Wno-register')
```